docs/resources/sagemaker_app_image_config.md

```diff
@@ -10,7 +9,77 @@ description: |-
 
 Resource Type definition for AWS::SageMaker::AppImageConfig
 
+## Example Usage
+
+To create a SageMaker app image config resource.
+
+Basic usage
+
+```terraform
+locals {
+  tags = [
+    {
+      key   = "Name"
+      value = "Sample IAM Role"
+    },
+    {
+      key   = "Environment"
+      value = "Development"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
+}
+
+resource "awscc_sagemaker_app_image_config" "example" {
+  app_image_config_name = "example"
+
+  kernel_gateway_image_config = {
+    kernel_specs = [{
+      name = "example"
+    }]
+  }
+  tags = local.tags
+}
+```
 
+Default File System Config
+
+```terraform
+locals {
+  tags = [
+    {
+      key   = "Name"
+      value = "Sample IAM Role"
+    },
+    {
+      key   = "Environment"
+      value = "Development"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
+}
+
+resource "awscc_sagemaker_app_image_config" "example" {
+  app_image_config_name = "example"
+
+  code_editor_app_image_config = {
```

Comment:
- `code_editor_app_image_config` doesn't have `file_system_config`, shouldn't it be under `kernel_gateway_image_config`?
  - Reference : https://registry.terraform.io/providers/hashicorp/awscc/latest/docs/resources/sagemaker_app_image_config#file_system_config
- Can we provide some config details than an empty config object ?

Reply:
@quixoticmonk Thanks for your review, changes has been done now.